auth-server/src/services/email.ts

```diff
@@ -76,7 +79,9 @@ export const sendResetPasswordEmail = (user: User, token: PasswordResetToken) =>
 		html: text
 	}
 
-	console.log(`Sending password reset email to ${user.email}`)
-
-	sgMail.send(msg);
+	try {
+		sgMail.send(msg);
```

Comment:
remove try  use .catch(error=> ...) otherwise you will have to convert function to async. we want this request to be running in background. function is returning promise